iDiff/cmd/iDiff.go

```diff
@@ -34,67 +32,22 @@ func validateArgs(args []string) (bool, error) {
 	if validArgNum, err := checkArgNum(args); !validArgNum {
 		return false, err
 	}
-	if validArgType, err := checkArgType(args); !validArgType {
-		return false, err
-	}
 	return true, nil
 }
 
 func checkArgNum(args []string) (bool, error) {
 	var errMessage string
 	if len(args) < 3 {
-		errMessage = "Too few arguments. Should have three: [IMAGE ID] [IMAGE ID] [DIFFER]."
+		errMessage = "Too few arguments. Should have three: [IMAGE] [IMAGE] [DIFFER]."
 		return false, errors.New(errMessage)
 	} else if len(args) > 3 {
-		errMessage = "Too many arguments. Should have three: [IMAGE ID] [IMAGE ID] [DIFFER]."
+		errMessage = "Too many arguments. Should have three: [IMAGE] [IMAGE] [DIFFER]."
 		return false, errors.New(errMessage)
 	} else {
 		return true, nil
 	}
 }
 
-func checkArgType(args []string) (bool, error) {
-	var buffer bytes.Buffer
-	valid := true
-	if !checkImageID(args[0]) {
-		valid = false
-		errMessage := fmt.Sprintf("Argument %s is not an image ID\n", args[0])
-		buffer.WriteString(errMessage)
-	}
-	if !checkImageID(args[1]) {
-		valid = false
-		errMessage := fmt.Sprintf("Argument %s is not an image ID\n", args[1])
-		buffer.WriteString(errMessage)
-	}
-	if checkImageID(args[2]) {
-		valid = false
-		buffer.WriteString("Do not provide more than two image IDs\n")
-	} else if !checkDiffer(args[2]) {
```

Comment:
I think we should try to keep some of the argument checking, especially the checkDiffer.